docs/ibexa_engage/install_ibexa_engage.md

```diff
@@ -1,10 +1,10 @@
 ---
-description: Install and configure Ibexa Engage.
+description: Install and configure Ibexa Engage
 edition: experience
 month_change: true
 ---
 
-# [[= product_name_engage =]]
+# Ibexa Engage
```

Comment:
What's the reasoning behind this change? Just curious 😄 Is it needed for the card title to be displayed properly?

Reply:
Yes, exactly :)